transports/quartz/src/test/resources/quartz-connector-hot-stop-restart-test.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<mule xmlns:quartz="http://www.mulesoft.org/schema/mule/quartz" xmlns="http://www.mulesoft.org/schema/mule/core"
+	xmlns:doc="http://www.mulesoft.org/schema/mule/documentation"
+	xmlns:spring="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans-current.xsd
+http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+http://www.mulesoft.org/schema/mule/quartz http://www.mulesoft.org/schema/mule/quartz/current/mule-quartz.xsd">
+
+	<quartz:connector name="Quartz" validateConnections="true" />
+
+	<flow name="testQuartzFlowFlow1">
+		<quartz:inbound-endpoint jobName="JustPingIt1"
+			repeatInterval="1000">
+			<quartz:event-generator-job />
+		</quartz:inbound-endpoint>
+		<logger level="INFO" message="quartz message from flow 1..." />
```

Comment:
Don't use loggers in the test config, use an echo-component if there is nothing to do